options/locale/locale_en-US.ini

```diff
@@ -425,7 +425,6 @@ update_theme = Update Theme
 update_profile = Update Profile
 update_profile_success = Your profile has been updated.
 change_username = Your username has been changed.
-change_username_prompt = Note: username changes also change your account URL.
```

Comment:
This is still true.

Additional entry should be added to explain that redirection will be created until someone else claims old name.

Reply:
If that's true, then one should be added for repository redirection as well.  This was removed for the same reason as https://github.com/go-gitea/gitea/pull/807#discussion_r99341636

Reply:
Look at GitHub notice when renaming.

Reply:
Ok, agreed.  So this prompt plus a new one, or just an alteration to this prompt?  Should I add in a new prompt for repository redirection too?

Reply:
>Should I add in a new prompt for repository redirection too?
In other PR

Reply:
>Ok, agreed. So this prompt plus a new one, or just an alteration to this prompt

However you see fit, but based on how long it is and styling we'll most likely need another one.